src/simperium/channel.js

```diff
@@ -445,6 +445,11 @@ Channel.prototype.onChangeVersion = function( data ) {
 };
 
 Channel.prototype.onVersion = function( data ) {
+	// invalid version, give up without emitting
+	if ( data.slice( -2 ) === '\n?' ) {
```

Comment:
Where does `\n?` come from?

Reply:
it's the server message - I should document this…